frame/staking/src/benchmarking.rs

```diff
@@ -792,12 +792,10 @@ benchmarks! {
 	}
 
 	get_npos_voters {
-		// number of validator intention.
-		let v in (MaxValidators::<T>::get() / 2) .. MaxValidators::<T>::get();
-		// number of nominator intention.
-		let n in (MaxNominators::<T>::get() / 2) .. MaxNominators::<T>::get();
-		// total number of slashing spans. Assigned to validators randomly.
-		let s in 1 .. 20;
+		// number of validator intention. we will iterate all of them.
+		let v in (MaxValidators::<T>::get() / 4) .. MaxValidators::<T>::get();
+		// number of nominator intention. we will iterate all of them.
+		let n in (MaxNominators::<T>::get() / 4) .. MaxNominators::<T>::get();
```

Comment:
Same here, not sure why this denominator? I can see that this lines up with MaxValidators as they are set to exactly same number for benches (100), but the general idea is not clear.